Dockerfile

```diff
@@ -29,6 +28,41 @@ RUN pip install -U pip \
 RUN bash -c "if [ $DEVEL_COLLECTION_LIBRARY -ne 0 ]; then \
     ansible-galaxy collection install ${DEVEL_COLLECTION_REPO} --force; fi"
 
+WORKDIR $APP_DIR
+COPY . $WORKDIR
```

Comment:
Can you use "$APP_DIR" instead? I don't remember the details but I remember issues with $WORKDIR variable which is not honored in some places.

Reply:
@Alex-Izquierdo I can but these two lines were there originally so I left them unchanged, only moved them. Should I change them then?

Reply:
Oh, that's true. I thought it was something new. Ok, if nobody has complained until now I think it is fine to leave them. :)